### PR TITLE
feat: fetch permits from database

### LINKED ISSUE #5: `audit` dApp: Show issues with unclaimed permits
Depends on https://github.com/ubiquity/pay.ubq.fi/issues/100, https://github.com/ubiquity/pay.ubq.fi/issues/110

Right now the [audit page](https://pay.ubq.fi/audit) only displays issues with claimed permits.

We should also show issues with unclaimed permits.

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves [#111](https://github.com/ubiquity/audit.ubq.fi/issues/5)

### CODE REVIEW

**Comment by gentlementlegen** [comment]
Got the error
```
Error: 🧨 Failed: Resource not accessible by integration
Error: 📚 Stack: HttpError: Resource not accessible by integration
    at file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:18847:26
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async createCheck (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:21997:[20](https://github.com/ubiquity/audit.ubq.fi/actions/runs/8120352504/job/22197453898?pr=2#step:5:21))
    at async createCheckId (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:22174:15)
    at async timeTask (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:22314:18)
    at async run3 (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:28085:17)
Error: HttpError: Resource not accessible by integration
```
in the build. Probably missing `permissions` in the Action file.

**Comment by 0x4007** [comment]
> Got the error
> 
> ```
> Error: 🧨 Failed: Resource not accessible by integration
> Error: 📚 Stack: HttpError: Resource not accessible by integration
>     at file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:18847:26
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async createCheck (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:21997:[20](https://github.com/ubiquity/audit.ubq.fi/actions/runs/8120352504/job/22197453898?pr=2#step:5:21))
>     at async createCheckId (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:22174:15)
>     at async timeTask (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:22314:18)
>     at async run3 (file:///home/runner/work/_actions/Codex-/knip-reporter/v2/dist/index.mjs:28085:17)
> Error: HttpError: Resource not accessible by integration
> ```
> 
> in the build. Probably missing `permissions` in the Action file.

@gitcoindev please prioritize this

**Comment by gentlementlegen** [comment]
It seems that there is a `permissions: write-all` in the Action. Maybe the settings of the repo take over, I cannot have access to them, but according to the logs the permissions were:
```
GITHUB_TOKEN Permissions
  Actions: read
  Checks: read
  Contents: read
  Deployments: read
  Discussions: read
  Issues: read
  Metadata: read
  Packages: read
  Pages: read
  PullRequests: read
  RepositoryProjects: read
  SecurityEvents: read
  Statuses: read
```
which does not provide write permissions.

**Review by rndquu — APPROVED** [review]
Knip CI is failing but I don't think it should block this PR from merging

File: static/index.html
Comment (0x4007) [inline static/index.html:1]: Formatting looks wrong here

**Comment by gitcoindev** [comment]
> Knip CI is failing but I don't think it should block this PR from merging

Yes, I agree, please ignore Knip CI failures for now. I will fix the Knip CI permission this week, after staking issue solution is finalized.

**Comment by 0x4007** [comment]
Merge conflict so can't merge but once that's resolved I think just merge

File: static/index.html
Comment (gentlementlegen) [reply static/index.html:1]: Fixed my indent settings that seemed different from the ones used in the project.

**Review by 0x4007 — APPROVED** [review]

**Pull request merged**
